install.sh

```diff
@@ -56,8 +56,9 @@ if ! sudo apt-get update; then
     die_cmd "Failed to update APT cache."
 fi
 
-if ! sudo apt-get install gpg wget; then
-    die_cmd "Failed to check if needed packages are installed."
+missing_dependencies="$( wget --version > /dev/null 2>&1 || echo wget ) $( gpg --version > /dev/null 2>&1 || echo gpg )"
+if ! ( test "$missing_dependencies" = " " || sudo apt-get install --mark-auto "$missing_dependencies" ); then
+    die_cmd "Failed to install needed packages."
```

Comment:
This still feels a bit hacky to me - the `dpkg-query` stuff below is what makedeb uses internally, perhaps we could just use it here with something like this?

```sh
mapfile -t installed_packages < <(dpkg-query -Wf '${Package}\n')
missing_dependencies=()

for pkg in 'wget' 'gpg'; do
  if ! printf '%s\n' "${installed_packages[@]}" | grep -q "${pkg}"; then
    missing_dependencies+=("${pkg}")
  fi
done
```